build-executor.sh

```
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+if [ -n "${DOCKER_LOGIN}" ] && [ -n "${DOCKER_PASSWORD}" ] && [ -n "${DOCKER_REGISTRY_URL}" ]; then
```

Shellcheck (0.10.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)